internal/cmd/operator-sdk/olm/cmd.go

```diff
@@ -15,13 +15,23 @@
 package olm
 
 import (
+	"fmt"
+
 	"github.com/spf13/cobra"
 )
 
+// Officially supported OLM versions by Operator SDK
+var (
+	OLMSupportedVersions = "unknown"
+)
+
```

Comment:
Is this supported version that we can INSTALL? or supported version that we work with?

Reply:
Its the supported version we work with. Thats why I added this from the doc - `Any other version installed with this command may work but is not officially tested`, to say that any version can be installed but only these are supported

Reply:
I'm thinking about this from a downstream perspective, where OLM will already be installed in the OpenShift cluster. So I'm wondering if I need to adjust these values in the downstream build to align with the OLM version that is shipped with OpenShift cluster or not.